packages/world/src/systems/MoveSystem.sol

```diff
@@ -14,15 +14,15 @@ import { PlayerUtils } from "../utils/PlayerUtils.sol";
 import { MoveLib } from "./libraries/MoveLib.sol";
 
 contract MoveSystem is System {
-  function move(EntityId caller, Vec3[] memory newCoords) public {
+  function move(EntityId caller, Vec3[] calldata newCoords) public {
```

Comment:
iirc using `calldata` args makes it much harder to call this from another system using the interface? like can't construct a new memory vec3 array and pass it in here?

Reply:
I think it should be easy, but iirc there is a bug in system libs that doesn't allow this so I'll update